### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by josephmisiti**
minor PR - added some directions on how to get started for people new to go

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,28 @@ This is a simple blogging engine with a few basic features:
...
 and bad design. It's a demonstration of the surface API of Authboss and how the library can be used to make a functioning web
 project, to use this sample as anything else is malpractice.
+
+# Getting Started
+
+To get started, clone the repo and execute the following commands
+
+```
+cd authboss-sample
+export GOPATH=`pwd`
```
Comment (aarondl) [inline README.md:23]: This isn't typical Go workflow. We don't use Godeps or Glide which would require this. As such I think the instructions should be modified to reflect that.

**Comment by josephmisiti** [comment]
ok - which workflow are you using ?

**Comment by aarondl** [comment]
Just the plain Go one. Where there's only a single GOPATH, so setting GOPATH isn't required (unless you're also showing people how to install Go).

**Comment by josephmisiti** [comment]
I am also using `go get` also, the only reason I am setting `GOPATH` is so the packages are installed locally rather than globally. I think it is basically the difference between `npm install X` and `npm install -g X`

**Comment by aarondl** [comment]
@josephmisiti Sorry, the e-mail notification for your latest comment got lost.

https://golang.org/doc/code.html
This is the workflow I describe. Notice they set GOPATH once and never again.

Your workflow doesn't work if you're building libraries, for example if you were to pull down authboss in this same manner and then try to use it from a different project on your local machine it would break because it's outside of a GOPATH and you would have to download it again. And if you are using it as a crude form of vendoring I'd highly suggest you try [godep](https://github.com/tools/godep) or [glide](https://github.com/Masterminds/glide) instead which use the Go 1.5 vendoring experiment to do the more correct thing.

If you use a single GOPATH with `$GOPATH/bin` in your `$PATH` as is standard outside tools like godep and glide; it's almost easier

``` bash
go get github.com/go-authboss/authboss-sample
cd $GOPATH/src/github.com/go-authboss/authboss-sample
./authboss-sample
```

**Comment by josephmisiti** [comment]
cool thank you !

**Comment by aarondl** [comment]
Would you like to update this pull request to match instructions like that? It was thanks to you that this issue will be addressed after all, should have your authorship in the git log :)

**Comment by aarondl** [comment]
PR is stagnant. Closing.

**Pull request closed without merging**